app/assets/stylesheets/_footer.scss

```diff
@@ -0,0 +1,44 @@
+footer {
+  right: 0;
+  bottom: 0;
+  left: 0;
+  padding: 10px;
+  background-color: #CCCCCC;
+  color: #808080;
+  height: 80px;
+  width: 100%;
+  font-size: .8rem;
+  text-align: center;
+  > p {
+    margin: 0;
+  }
+}
+.footer__nav {
+  list-style: none;
+  margin-top: 10px;
+  margin-bottom: 0;
+  padding-left: 0;
+  padding-bottom: 10px;
+  > li {
+    display: inline-block;
+    padding: 4px;
+    font-size: 1rem;
+    line-height: 1.2rem;
+    color: #808080;
+    &:after {
+        content: "\00a0\00a0\00a0|";
+      }
+      &:last-of-type:after {
+        content: none;
+      }
+  }
+    > li > a {
+      color: inherit;
```

Comment:
I've been doing it this way because every time I try to nest it, the behavior is not as I expect, but placing it after closing out the li and just doing the path to it (> li > a { ) seems to make everything work how I want.